src/org/usfirst/frc/team467/robot/TalonSpeedControllerGroup.java

```diff
@@ -0,0 +1,192 @@
+package org.usfirst.frc.team467.robot;
+
+import org.apache.log4j.Logger;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.SpeedController;
+
+public class TalonSpeedControllerGroup implements SpeedController {
+	private static final Logger LOGGER = Logger.getLogger(TalonSpeedControllerGroup.class);
+	WPI_TalonSRX leader;
+	WPI_TalonSRX follower1;
+	WPI_TalonSRX follower2;
+	
+	private int previousSensorPosition;
+	
+	ControlMode controlMode = ControlMode.PercentOutput;
+	
+	public TalonSpeedControllerGroup() {
```

Comment:
Is this constructor ever called?

Reply:
NO, but it allows for the creation of a Controller Group without any motors or wheels, no function of the class will be able to be run; Just for efficiency purposes